data/init.sql

```diff
@@ -48,6 +48,25 @@ CREATE TABLE IF NOT EXISTS discussion_messages (
 	updated_at TIMESTAMP
 );
 
+<<<<<<< HEAD
+
+CREATE TABLE IF NOT EXISTS meetings (
+	id INT(11) UNSIGNED AUTO_INCREMENT PRIMARY KEY,
+	group_id INT(11),
+	user_id INT(11),
+	title VARCHAR(60),
+	agenda VARCHAR(1000),
+	minutes VARCHAR(10000),
+	planned_date DATE, 
+	planned_time TIME,
```

Comment:
is there a specific reason why this isn't just a timestamp?

Reply:
because this is being entered by the user, and that way I can use the html date and time input options which look nice